Buildplan/Dockerfile-Jenkins

```
@@ -67,9 +67,8 @@ ENV COMPOSER_H_DIR="${HOME}/.composer"
 ENV PLUGIN_DIR="${WP_PLUGINS_DIR}/smartling-connector"
 ENV LOCAL_GIT_DIR="/plugin-dir"
 
-COPY Buildplan/release.sh /release.sh
-COPY Buildplan/test.sh /test.sh
-RUN chmod +x /release.sh /test.sh
+COPY Buildplan/release.sh Buildplan/releaseVersionCheck.sh Buildplan/test.sh /
+RUN chmod +x /release.sh /releaseVersionCheck.sh /test.sh
```

Or use wildcard since it's our folder with limited number of files
```suggestion
RUN chmod +x /*.sh
```